Automattic/vip-cli
# Bump dependencies

via `npm audit fix`. Should address the security alerts.

## To Test

- Checkout branch
- `npm install`
- `npm run build`
- Verify build worked.
